__tests__/lint/assertions-custom-function-options/plugin.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  id: 'local',
+  assertions: {
+    checkLength: (value, opts, location) => {
+      if (value.length < opts.min) {
+        return { isValid: false, location };
+      }
+      return { isValid: true };
+    },
+  },
+};
```

Comment:
We would need some docs on how to write this custom function, what the arguments are, what the expected return values are, and an example too.